src/components/atoms/SubmittedQuestion/SubmittedQuestion.tsx

```diff
@@ -0,0 +1,16 @@
+'use client';
+import Image from 'next/image';
+import icon from '@public/person.png';
+
+type SubmittedQuestionType = {
+  message: string;
+};
+
+export const SubmittedQuestion = ({ message }: SubmittedQuestionType) => {
+  return (
+    <div className="rounded-md text-user_grey sm:text-sm w-full text-left p-3 flex items-center gap-3">
```

Comment:
do we want the image for the response/question to be centered? or at the top of the response